logging/performance_reliability/logging-input-spec-filtering.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-input-spec-filtering"]
+= Filtering logs by metdata
+:context: logging-input-spec-filtering
+
+toc::[]
+
+
+You can filter logs in the `ClusterLogForwarder` CR to select or ignore the entire log stream based on metadata using the `input` selector. As an administrator, you can include or exclude log collection according to your interest. As a developer, you can include or exclude log collection to reduce the memory and CPU load on the collector. The following options are available for log filtering:
+
+[IMPORTANT]
+====
+The Vector collector must be set up in your logging deployment to use this feature. The Fluentd collector does not support the `input` spec filtering in the `ClusterLogForwarder` CR.
```

Comment:
nit:   I think we will need to mention our old collector less and less going forward and my preference would be not mention fluentd in this case?   Something as simple as "Vector collector only" may be sufficient here.

Reply:
@cahartma Agreed. Removed the second line that mentions fluentd.